JustSaying/JustSaying.nuspec

```diff
@@ -8,6 +8,6 @@
     <owners>JUST EAT plc</owners>
```

Comment:
No, maybe. I think I'll need to keep it to easily include multiple assemblies in a single nupkg (there's an issue around this) I am conflicted about if we want to do this.

If we need it, I can reference it in the csproj (there's a new property to link to nuspecs), and dotnet pack will do some magic. But I'd like to review the options.